doc-content/enterprise-only/vehicle-route-planning/run-vrp-manually-proc.adoc

```diff
@@ -0,0 +1,91 @@
+* Internet access is available.
+
+.Procedure
+. Download routing data.
++
+The routing engine requires geographical data to calculate the time it takes vehicles to travel between locations.
+You must download and store OpenStreetMap (OSM) data files on the local file system before you run OptaWeb Vehicle Routing.
++
+NOTE: The OSM data files are typically between 100 MB to 1 GB and take time to download so it is a good idea to download the files before building or starting the OptaWeb Vehicle Routing application.
+
+.. Open `http://download.geofabrik.de/` in a web browser.
+.. Click a region in the *Sub Region* list, for example *Europe*. The subregion page opens.
+.. In the *Sub Regions* table, download the OSM file (`.osm.pbf`) for a country, for example Belgium.
+
+. Create the data directory structure.
++
+OptaWeb Vehicle Routing reads and writes several types of data on the file system.
+It reads OSM (OpenStreetMap) files from the `openstreetmap` directory, writes a road network graph to the `graphhopper` directory, and persists user data in a directory called `db`.
+Create a new directory dedicated to storing all of these data to make it easier to upgrade to a newer version of OptaWeb Vehicle Routing in the future and continue working with the data you created previously.
+
+.. Create the `$HOME/{VRP-DATA-DIR}` directory.
+.. Create the `openstreetmap` directory in the `$HOME/{VRP-DATA-DIR}` directory:
++
+[source,subs="attributes+"]
+----
+$HOME/{VRP-DATA-DIR}
+└── openstreetmap
+----
+
+.. Move all of your downloaded OSM files (files with the extension `.osm.pbf`) to the `openstreetmap` directory.
++
+The rest of the directory structure is created by the OptaWeb Vehicle Routing application when it runs for the first time.
+After that, your directory structure is similar to the following example:
++
+[source,subs="attributes+"]
+----
+$HOME/{VRP-DATA-DIR}
+
+├── db
+│   └── vrp.mv.db
+├── graphhopper
+│   └── belgium-latest
+└── openstreetmap
+    └── belgium-latest.osm.pbf
+----
+// TODO maybe replace this with a screenshot, doesn't look good in PDF.
+. Change directory to `{URL_COMPONENT_VRP}-distribution-{MAVEN_ARTIFACT_VERSION}/sources/optaweb-vehicle-routing-standalone/target`.
+. To run OptaWeb Vehicle Routing, enter the following command:
++
+[source,subs="attributes+"]
+----
+java -jar optaweb-vehicle-routing-standalone-{MAVEN_ARTIFACT_VERSION}.jar \
+--app.persistence.h2-dir=$HOME/{VRP-DATA-DIR}/db \
+--app.routing.gh-dir=$HOME/{VRP-DATA-DIR}/graphhopper \
+--app.routing.osm-dir=$HOME/{VRP-DATA-DIR}/openstreetmap \
+--app.routing.osm-file=<OSM_FILE_NAME> \
+--app.region.country-codes=<COUNTRY_CODE> \
+----
+In this command, replace the following variables:
+
+* `<OSM_FILE_NAME>`: The OSM file for the region that you want to use and that you downloaded previously
+* `<COUNTRY_CODE>`: The code for the region that you want to use
```

Comment:
'<COUNTRY_CODE> ' needs to be changed to '<COUNTRY_CODE_LIST>: A comma-separated list of country codes used to filter geosearch queries. For a list of country codes, see ISO 3166 Country Codes. '